src/transformation/builtins/math.ts

```diff
@@ -32,6 +33,13 @@ export function transformMathCall(context: TransformationContext, node: Property
     const params = transformArguments(context, node.arguments, signature);
 
     const expressionName = expression.name.text;
+
+    if (context.options.luaTarget === LuaTarget.Lua53 && expressionName === "atan2") {
```

Comment:
I'd prefer if you handle all atan2 cases in the same place (probably in its switch case)